image_writers_guide/sti.adoc

```diff
@@ -163,6 +163,24 @@ EOF
 ----
 ====
 
+[[using-the-sti-environment-file]]
+== Using the STI environment file
+
+STI has a capability to set environment values specified in the
+`.sti/environment` file, that exists inside the application source repository.
```

Comment:
True, reworked suggestion is:
STI has a capability to set environment values, specified in the `.sti/environment` file inside your application, those variables will be then present during the build process and in the final docker image.